scripts/external.test

```diff
@@ -13,6 +13,18 @@ if [ $? -ne 0 ]; then
 
     # cloudflare seems to change CAs quickly, disabled by default
     if test -n "$WOLFSSL_EXTERNAL_TEST"; then
+
+	BUILD_FLAGS="$(./examples/client/client '-#')"
+	if echo "$BUILD_FLAGS" | fgrep -q -e ' -DWOLFSSL_SNIFFER '; then
+	    echo 'skipping WOLFSSL_EXTERNAL_TEST because -DWOLFSSL_SNIFFER configuration of build is incompatible.'
```

Comment:
I will look into this separately. This test should work with the sniffer build option enabled. The cipher suite can be provided explicitly for the external test and it won't use the static RSA cipher suite.